gcc/rust/backend/rust-compile-expr.h

```diff
@@ -354,7 +354,7 @@ class CompileExpr : public HIRCompileBase
   void visit (HIR::NegationExpr &expr) override
   {
     auto op = expr.get_expr_type ();
-    auto negated_expr = CompileExpr::Compile (expr.get_expr (), ctx);
+    auto negated_expr = CompileExpr::Compile (expr.get_expr ().get (), ctx);
```

Comment:
Some day I hope we refactor these to take a reference to the unique_ptr to enforce the ownership or take an lvalue ref to the object instead of the pointer. This might be something you can help me with later on if you have time.

Reply:
yeah, sure. :)